components/Chat/Message/V3Message.tsx

```
@@ -1,116 +1,738 @@
+      return null;
+    }
+
+    if (isReactionMessage(message)) {
+      logger.debug(`isReactionMessage message`);
+      return null;
+    }
+
+    // Need DecodedMessageAllTypes to work
+    // const _ensureNever: never = message;
+    throw new Error("Unknown message type");
+    return null;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)